src/content/en/resources/translations.md

```diff
@@ -21,7 +21,7 @@ your contribution:
 1. Find the document you are going to translate.
```

Comment:
don't forget to update `{# wf_updated_on: YYYY-MM-DD #}`

Reply:
Should it be the updated date of the translation or should it align with
the original article?